README.md

```diff
@@ -0,0 +1,7 @@
+Electrum flatpak buildfile
```

Comment:
This README is outdated (wrong id/filename).
Moreover, Flathub maintainers don't like README files in general.